assets/scss/templates/_dashboard.scss

```diff
@@ -5,4 +5,142 @@
 ////
 
 .dp-library {
+  .dp-dashboard-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .dp-dashboard-title-link {
+    font-size: $dp-sizing--lvl1;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    &::before {
+      width: 15px;
+      height: 16px;
+      margin-right: 5px;
+      background: url('../img/dashboard-list-icon-title.svg') no-repeat 0 0;
+      content: '';
+    }
+  }
+  .dp-dashboard-panel {
+    margin-bottom: 80px;
+    position: relative;
+
+    .dp-overlay {
+      position: absolute;
+      background-color: $dp-color-ghostwhite;
+      width: calc(100% - 24px);
+      height: 100%;
+      align-items: center;
+      justify-content: center;
+      opacity: 0.8;
+      border: 2px solid $dp-color-silver;
+      border-radius: 3px;
+      margin: 0 12px;
+      font-size: 16px;
+      color: $dp-color-darkgrey;
+      display: none;
+      &.show {
+        display: flex;
+      }
+    }
+
+    .dp-dashboard-kpi {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      border: 2px solid $dp-color-snow;
+      padding: 30px;
+      border-radius: 3px;
+    }
+    .dp-kpi-value {
+      font-size: $dp-sizing--lvl7;
+      line-height: $dp-sizing--lvl7;
+      padding-bottom: $dp-spaces--lv2;
+    }
+    .dp-kpi-title {
+      font-size: $dp-sizing--lvl2;
+      line-height: $dp-sizing--lvl2;
+    }
+    .dp-kpi-period {
+      font-size: $dp-sizing--lvl1;
+      line-height: $dp-sizing--lvl5;
+      text-transform: uppercase;
+    }
+    .dp-bold {
+      font-weight: bold;
+    }
+    &.disabled {
+      .dp-dashboard-icon.deliveries {
+        background-image: url('../img/dashboard-deliveries-disabled.svg');
```

Comment:
These icons cannot be part of icon-font right? for its complexity I mean.

Reply:
These icons will be part of an ico-font library when we iplement it in the UI Library. 😉